docs/setup.md

```diff
@@ -13,72 +13,67 @@
 
 ## Prerequisites
 
-The **Continuous Delivery** solution pack depends on the following solution packs that are installed automatically &ndash; if not already installed.
+The **Continuous Delivery** solution pack depends on the following solution packs. FortiSOAR automatically installs these dependencies if missing.
 
 | Name           | Type          | Version | Purpose                                          |
 |:---------------|:--------------|:--------|:-------------------------------------------------|
 | SOAR Framework | Solution Pack | v2.0.0  | Required for Incident Response modules           |
 | GitHub         | Connector     | v1.2.0  | Required for source control through GitHub       |
 | GitLab         | Connector     | v2.0.0  | Required for source control through GitLab       |
-| CICD Utils     | Connector     | v1.1.0  | Required for performing CICD specific operations |
+| CICD Utils     | Connector     | v1.1.0  | Required for performing Continuous Delivery specific operations |
```

Comment:
Add widgets also or removes the connector list and use solution pack word in above comment